src/cli/cli.ts

```diff
@@ -58,59 +92,150 @@ async function translate() {
     error(messages.file.no_file_in_path);
     return;
   }
+  
+  
+  let translator = myOpts.translator
+  if (translator && translator !== '') {
+    translator = capitalize(myOpts.translator)
```

Comment:
I don't think that we need capitalization there, forcing a user to enter valid name seems better for me.

Reply:
When typing the option value in lowercase letters, an error message is displayed along with the selection menu again. Personally, I find this illogical. Anyway, thank you for alerting me to these errors. I will work on fixing them all tomorrow. I invite you to download the full version and try it out. Check more